lib/Serializer.js

```diff
@@ -44,11 +98,161 @@ class Serializer {
     };
   }
 
+  /* The characters with a highly unlikely chance of occurrence in strings, alone or in combination. */
+  _bigIntMarkChars = ['෴', '߷', '֍'];
+
+  /* Generates an array of all combinations of `_bigIntMarkChars` with the requested length. */
```

Comment:
I know these are unlikely, but is there a more reliable way to have a BOM?

Reply:
It doesn't just use these guys. It creates a combination of 1 to Infinity of them. So even if these guys are all present, there is always a sequence that is long enough to not be present.

A future enhancement could dynamically generate a large collection based on ranges and experiment with them.

Reply:
I'm probably slow. These are all numbers. You get things like `෴18014398509481982`, why not `N:18014398509481982`?